packages/stateManager/tests/stateManager.test.ts

```diff
@@ -231,6 +231,12 @@ describe("CRUD functions update stores", async () => {
         chainId: 1,
         tokenAddr: zeroAddress,
       },
```

Comment:
these updates should be separate sub-tests so we can see which/where it fails but as discussed, separate PR is 👌🏻